src/rp2_common/hardware_flash/include/hardware/flash.h

```diff
@@ -44,6 +33,22 @@
  * \include flash_program.c
  */
 
+// PICO_CONFIG: PARAM_ASSERTIONS_ENABLED_FLASH, Enable/disable assertions in the flash module, type=bool, default=0, group=hardware_flash
+#ifndef PARAM_ASSERTIONS_ENABLED_FLASH
+#define PARAM_ASSERTIONS_ENABLED_FLASH 0
+#endif
+
+#define FLASH_PAGE_SIZE (1u << 8)
+#define FLASH_SECTOR_SIZE (1u << 12)
+#define FLASH_BLOCK_SIZE (1u << 16)
+
+#define FLASH_UNIQUE_ID_SIZE_BYTES 8
+
+// PICO_CONFIG: PICO_FLASH_SIZE_BYTES, size of primary flash in bytes, type=int, default=-1, group=hardware_flash
```

Comment:
Probably ought to remove the default from this line too?
(I've just checked, and all the scripts/programs dealing with `pico_configs.tsv` cope fine with `default` being absent)

Reply:
#183 enforces this :slightly_smiling_face:

Reply:
You still need to fix the PICO_CONFIG line to remove the `default`, i.e.
```
// PICO_CONFIG: PICO_FLASH_SIZE_BYTES, size of primary flash in bytes, type=int, group=hardware_flash
```